cmd/qat_plugin/qat_plugin_test.go

```diff
@@ -31,20 +32,30 @@ func init() {
 	debug.Activate()
 }
 
-func createTestFiles(prefix string, dirs []string, files map[string][]byte) error {
+func createTestFiles(prefix string, dirs []string, files map[string][]byte, symlinks map[string]string) error {
 	for _, dir := range dirs {
+
 		err := os.MkdirAll(path.Join(prefix, dir), 0755)
 		if err != nil {
 			return errors.Wrap(err, "Failed to create fake device directory")
 		}
+
 	}
 	for filename, body := range files {
+
 		err := ioutil.WriteFile(path.Join(prefix, filename), body, 0644)
 		if err != nil {
 			return errors.Wrap(err, "Failed to create fake vendor file")
 		}
+
 	}
+	for link, target := range symlinks {
 
+		os.MkdirAll(path.Join(prefix, target), 0755)
+		cmd := exec.Command("ln", "-s", path.Join(prefix, target), path.Join(prefix, link))
+		cmd.Run()
```

Comment:
There's a way to create symlinks in Go without calling creating external process. Use `os.Symlink()`.

Also it wouldn't harm to check errors after `os.MkdirAll()` and `os.Symlink()`.